Formula/fastlane.rb

```diff
@@ -0,0 +1,27 @@
+class Fastlane < Formula
+  desc "Easiest way to build and release mobile apps"
+  homepage "https://fastlane.tools"
+  url "https://github.com/fastlane/packaged-fastlane/archive/2.0.0.tar.gz"
+  version "2.137.0"
+  sha256 "5065f68499823f0137a848af7b975f4fccb4085a366f4fc2cfc27ba7e0188d41"
+  head "https://github.com/fastlane/packaged-fastlane.git"
+
+  depends_on "ruby@2.5"
+
+  def install
+    system "./install", version.to_s, prefix
```

Comment:
What does the `bundle-env` here do? It looks like https://github.com/fastlane/packaged-fastlane/blob/master/install#L14 is basically just doing a `gem install`; it'd be preferable to download the source repository and build it from there (so the downloads are cached). See `wpscan.rb` for an example of building a gem and installing it.